src/utils/rest-api.js

```diff
@@ -2428,6 +2424,59 @@ export function fetchMapEquipments(
     return backendFetchJson(fetchEquipmentsUrl);
 }
 
+export function fetchSingleEquipment(
```

Comment:
This function is not called

Reply:
yes, and its the same than fetchEquipmentInfos
I'll remove